adrian/script.js

```diff
@@ -0,0 +1,25 @@
+//1
+const multTwoNumbers = (num1, num2) => num1 * num2;
+
+//2
+let arrayOfNumbers = [3, 5, 67, 12, 34, 4, 9, 30];
+
+let filteredArray = arrayOfNumbers.filter((num) => num % 2 !== 0);
```

Comment:
Task 2: Multiply each odd number by 2.

Reply:
I did it but deleted it to check something and then forgot to add the .map(). Thanks, done!.

Reply:
Good job thanks